### PR TITLE
Error sphinx windows

### PR DESCRIPTION
**Pull request by pedrog99**

### CODE REVIEW

File: skfda/representation/_functional_data.py [hunk 1]
```diff
@@ -1163,22 +1163,25 @@ def take(  # noqa: WPS238
...
-                Indices to be taken.
+            indices: Indices to be taken.
             allow_fill: How to handle negative values in `indices`.
-                * False: negative values in `indices` indicate positional
-                  indices from the right (the default). This is similar to
-                  :func:`numpy.take`.
-                * True: negative values in `indices` indicate
-                  missing values. These values are set to `fill_value`. Any
-                  other negative values raise a ``ValueError``.
+            * False: negative values in `indices` indicate positional
```
Comment (vnmabus) [inline skfda/representation/_functional_data.py:1172]: This is wrongly represented as an additional parent in the generated documentation.

**Review by vnmabus — CHANGES_REQUESTED** [review]

File: skfda/preprocessing/feature_construction/_function_transformers.py [hunk 2]
```diff
@@ -212,19 +212,19 @@ class NumberCrossingsTransformer(
...
-                [-0.04652559],
-                [ 0.25396879],
-                [ 0.26095573],
-                [ 0.01042895],
-                [-0.22089135],
-                [-0.2074856 ],
-                [ 0.0126612 ],
-                [ 0.20089319],
-                [ 0.17107348]]])
+        [ 0.73041066],
```
[the remark this answers is not in the document]
Comment (pedrog99) [reply skfda/preprocessing/feature_construction/_function_transformers.py:215]: In Windows 10, I got unindentation Sphinx error with the previous format.

File: conftest.py [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
     np.set_printoptions(sign=' ')
 except TypeError:
     pass
 
+"""
+I introduced this change in order to adapt it to Windows 10
+operating system.
+More information about this problem in this GitHub issue:
+https://github.com/jupyter/jupyter-sphinx/issues/171#issuecomment-766953182
+"""
```
Comment (vnmabus) [inline conftest.py:18]: This is still a string instead of a comment (use `#`).

File: conftest.py [hunk 4]
```diff
@@ -1,12 +1,25 @@
...
+if (sys.version_info[0] == 3 and sys.version_info[1] >= 8
+        and sys.platform.startswith('win')):
```
Comment (vnmabus) [inline conftest.py:20]:
```suggestion
if (
    sys.version_info[0] == 3 and sys.version_info[1] >= 8
    and sys.platform.startswith('win')
):
```
Try to follow our style guide if possible.

**Review by vnmabus — CHANGES_REQUESTED** [review]

**Review by vnmabus — APPROVED** [review]
LGTM

**Pull request merged**